package.json

```diff
@@ -31,5 +31,17 @@
         "eslint-config-airbnb-base": "^12.1.0",
         "eslint-plugin-import": "^2.9.0",
         "jest": "^22.4.2"
+    },
+    "resolutions": {
+        "mem": "^5.1.1",
+        "lodash": "^4.17.15",
+        "stringstream": "^1.0.0",
+        "js-yaml": "^3.13.1",
+        "just-extend": "^4.0.2",
```

Comment:
The jump from `1.1.27` is likely to break things. Can we fix this by bumping `aws-sdk-mock` instead?

Chain: `aws-sdk-mock` -> `nise` -> `just-extend`